sqlmesh/utils/__init__.py

```diff
@@ -45,6 +48,19 @@ def random_id() -> str:
     return uuid.uuid4().hex
 
 
+def time_uuid(k: int = 4) -> str:
```

Comment:
this is not uuid though

Reply:
why do we need this exactly?

Reply:
ok, I understand why we need this, but we shouldn't call this uuid I think

Reply:
it is due to the k

it's possible that two files are created at the exact same time, in that case you need to sort by this as a backup

Reply:
what do you want to call it

Reply:
I think I understand why we need this, but we shouldn't be calling this uuid.

Reply:
maybe just `lock_id`? I feel like this logic is quite specific to the lock implementation. We can move it accordingly too